src/components/Sidebar/Sidebar.css

```diff
@@ -0,0 +1,109 @@
+.sidebar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+}
+.sidebar .menu_item {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  border-top: 1px solid #1b64ab;
+  color: #fff;
+}
+.sidebar ul li:hover {
+  background-color: #00407a;
+}
+.sidebar .menu_footer {
+  color: #fff;
+  margin-top: auto;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+.sidebar .menu_footer ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+.sidebar .menu_footer ul li {
+  display: flex;
+  align-items: left;
+  padding: 16px 24px;
+}
+.sidebar .text {
+  margin-left: 12px;
+  font-family: "Open Sans-Regular", Helvetica;
+  font-size: 1.2rem;
+  font-weight: 100;
+  letter-spacing: 1.94px;
+  line-height: 2.5rem;
+}
+.sidebar .icon svg path {
+  stroke: #fff;
+}
+.sidebar .menu_footer ul li .icon {
+  width: 24px;
+  height: 26px;
+  color: #fff;
+}
+.sidebar-collapsed .menu_footer ul li .text {
+  visibility: hidden;
+}
+
+.sidebar-full {
+  background: var(
+    --main-gradient,
+    linear-gradient(225deg, #005eaa 0%, #26418f 100%)
+  );
+  position: fixed;
+  width: 204px;
+  height: 100%;
+  flex: 0 0 calc(10px + 100px);
+  display: flex;
+  flex-direction: column;
+}
+.sidebar-full .menu_header {
+  padding: 8px 16px;
+}
+.sidebar-full .menu_header button {
+  width: 28px;
+  height: 24px;
+  padding: 0;
+  margin: 8px;
+  font-size: 10px;
+}
+.sidebar-full .menu_item {
+  height: 100%;
+}
+.sidebar .menu_item ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+.sidebar .menu_item ul li {
+  display: flex;
+  align-items: left;
+  padding: 16px 24px;
+}
+
+.sidebar-full .menu_item h2 {
+  font-size: 16.15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 120%;
+  letter-spacing: 1.938px;
+  text-transform: uppercase;
+  padding: 0 24px;
+}
+.sidebar-collapsed {
+  background: var(
+    --main-gradient,
+    linear-gradient(225deg, #005eaa 0%, #26418f 100%)
+  );
+  width: 80px;
+}
+.sidebar-collapsed .menu_item h2 {
+  visibility: hidden;
+}
+.sidebar-collapsed .menu_item ul li .text {
+  visibility: hidden;
+}
```

Comment:
I think we should we add the class `cdc-react` to the top-level sidebar component classes.

Reply:
This also doesn't seem to have media queries for the mobile ui/ux design.